OpenEdX/DI/ScreenAssembly.swift

```diff
@@ -177,15 +179,17 @@ class ScreenAssembly: Assembly {
                 interactor: r.resolve(DashboardInteractorProtocol.self)!,
                 connectivity: r.resolve(ConnectivityProtocol.self)!,
                 analytics: r.resolve(DashboardAnalytics.self)!,
-                config: r.resolve(ConfigProtocol.self)!
+                config: r.resolve(ConfigProtocol.self)!,
+                storage: r.resolve(CoreStorage.self)!
             )
         }
         
         container.register(AllCoursesViewModel.self) { r in
             AllCoursesViewModel(
                 interactor: r.resolve(DashboardInteractorProtocol.self)!,
                 connectivity: r.resolve(ConnectivityProtocol.self)!,
-                analytics: r.resolve(DashboardAnalytics.self)!
+                analytics: r.resolve(DashboardAnalytics.self)!,
```

Comment:
I don't know the reason for forced unwrapping here but it can be dangerous and can produce crashes.

Reply:
Thank you for your observation. While your point about the dangers of forced unwrapping is theoretically valid, in practice, it is crucial for us that the application crashes if Swinject Dependency Injection fails. Essentially, if the DI container fails to resolve a dependency, the application would be non-functional, and it's better to know this immediately through a crash rather than running with missing dependencies. This ensures that we address the issue promptly and maintain the integrity of the application's behavior.

Reply:
Make sense for now, i would suggest using `guard` here in future with some error alert instead of crashing the app.